order-service/pom.xml

```diff
@@ -251,12 +251,9 @@
                 <configuration>
                     <image>
                         <name>dockertmt/mmv2-${project.artifactId}:${project.version}</name>
-			<env>
-                            <!-- Install the gcc library with libstdc++ to support RocksDB -->
-                            <BP_APT_PACKAGES>libstdc++6 libgcc1 zlib1g libc6-dev libnuma1</BP_APT_PACKAGES>
-                        </env>    
+                        <builder>docker.io/paketobuildpacks/builder-jammy-base</builder>
```

Comment:
@coderabbitai, what is default builder used in spring boot 3.2 and spring boot 3.4 version